scanner/updater/rhel/parser.go

```diff
@@ -52,7 +52,22 @@ func (u *Updater) Parse(ctx context.Context, r io.ReadCloser) ([]*claircore.Vuln
 		if u.shouldSkipDefType(defType) {
 			return vs, nil
 		}
-
+		// Go look for the vuln name in the references when CVE, or parse the title when
+		// RHSA, RHBA or RHEA.
+		var name string
+		switch defType {
+		case ovalutil.CVEDefinition:
+			for _, r := range def.References {
+				if r.Source == "CVE" {
+					name = r.RefID
+					break
+				}
+			}
+		case ovalutil.RHBADefinition, ovalutil.RHSADefinition, ovalutil.RHEADefinition:
+			name = strings.TrimSpace(def.Title[:strings.Index(def.Title, ": ")])
+		default:
+			name = def.Title
+		}
```

Comment:
to guarantee compatibility and to simplify this, you can mimic what scanner v2 does:

```
func name(definition oval.Definition) string {
	if len(definition.References) > 0 {
		return definition.References[0].RefID
	}

	return "" // <- maybe replace this with def.Title, though
}
```